test/resttests.ts

```diff
@@ -176,5 +177,27 @@ describe('Rest Tests', function () {
         assert(restRes.result.args.baz === 'top');
     });
 
+    //
+    // getUrl path tests
```

Comment:
you're missing this test.

'/get/foo', 'http://httpbin.org/bar'

should result in 'http://httpbin.org/get/foo'